docs/api/security/psa_protected_storage.md

```diff
@@ -0,0 +1,19 @@
+## PSA Protected Storage
+
+### Description
+
+PSA Protected storage APIs allows saving and retrieving data from PSA protected storage.
+
+Unlike [PSA Internal storage](./psa_internal_storage.md), PSA Protected storage implemented to always run in NSPE side and redirect the calls to KVStore instance.
+
+TODO: Danniel Benor - review and add links to this page from Storage landing page 
```

Comment:
@dannybenor - please review this and previous file and links to them from the storage page

Reply:
@alzix, in my notes I wrote that this would be under APIs > Storage.
Is this the correct location for this document?

Reply:
IMO the `.md` file location is not matter as the rendered HTML page URL is set manually.
But i have no problem moving it.
Storage landing page still needs to be updated to include links to these files